etc/profile-a-l/ghostwriter.profile

```diff
@@ -47,14 +54,15 @@ seccomp !chroot
 seccomp.block-secondary
 #tracelog -- breaks
 
-private-bin context,gettext,ghostwriter,latex,mktexfmt,pandoc,pdflatex,pdfroff,prince,weasyprint,wkhtmltopdf
+private-bin affixcompress,analyze,chmorph,cmark,context,gettext,ghostwriter,hunspell,hunzip,hzip,latex,makealias,mktexfmt,munch,multimarkdown,pandoc,pdflatex,pdfroff,prince,unmunch,weasyprint,wkhtmltopdf,wordforms,wordlist2hunspell
 private-cache
 private-dev
 # passwd,login.defs,firejail are a temporary workaround for #2877 and can be removed once it is fixed
-private-etc @tls-ca,@x11,dbus-1,firejail,gconf,host.conf,mime.types,rpc,services,texlive
+private-etc @tls-ca,@x11,dbus-1,gconf,host.conf,mime.types,rpc,services,texlive
 private-tmp
 
 dbus-user filter
+dbus-user.talk ca.desrt.dconf
```

Comment:
It is a Qt program. Why alloe dconf?

Reply:
Nice catch, thanks! Probably I mixed things up with the peek profile. Anyway, I took it out.